src/test/kotlin/no/nav/familie/ef/sak/config/PdlClientConfig.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.familie.ef.sak.no.nav.familie.ef.sak.config
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.familie.ef.sak.integration.PdlClient
+import no.nav.familie.ef.sak.integration.dto.pdl.*
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Primary
+import org.springframework.context.annotation.Profile
+
+@Configuration
+@Profile("mock-pdl")
+class PdlClientConfig {
+
+    @Bean
+    @Primary
+    fun pdlClient(): PdlClient {
```

Comment:
Det som blir litt spesielt her er att denne bean kommer å bli brukt for alle unittester som bruker `OppslagSpringRunnerTest` ettersom den bruker `ApplicationLocal`

Reply:
Hmmm, trodde det var meningen at testene skulle ha eget oppsett og legge inn de mock-profilene de trengte selv.